src/elements/MenuDrawer/SubMenu/SubMenu.tsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import { Text } from 'react-native';
+
+import styles from './SubMenu.styles';
+
+const SubMenu = props => <Text style={{ ...styles.subMenu, marginBottom: 10 }}>{props.text}</Text>;
```

Comment:
Did you mean to leave the `marginBottom` styling outside of the `subMenu` obj?

Reply:
Oh no, I did not! Oops, I'll include it into subMenu 😄